src/issue/screens/issue.screen.js

```diff
@@ -230,7 +230,7 @@ class Issue extends Component {
       navigation,
     } = this.props;
 
-    const isLoadingData = isPendingComments || isPendingIssue;
+    const isLoadingData = isPendingComments || isPendingIssue || false;
```

Comment:
I don't think false is needed here. It will only use that value if the other two values are false.